src/plguins/useCellRangeSelection.js

```diff
@@ -211,24 +211,10 @@ function useInstance (instance) {
     [allColumns, cellsById, cellIdSplitBy, rows]
   )
 
-  // const selectedFlatCells = React.useMemo(()=> {
-  //   const selectedFlatCells = []
-  //   rows.forEach(row=> {
-  //     row.allCells && row.allCells.forEach(cell => {
-  //       cell.isSelected = !!selectedCellIds[cell.id]
```

Comment:
Tried to add `selectedFlatCells` with `cell.isSelected`.

`selectedFlatCells ` had `cell.isSelected` in it, but while we loop `row.cells` on UI file (app.js), it was missing